R-notebooks/dashboard_functions.R

```diff
@@ -20,26 +20,31 @@ make_covidcast_signal <- function(destination, source, geo_type) {
   return(destination)
 }
 
-plot_28_day_unique <- function(df_to_plot, plural_geo_title) {
+plot_unique_geo_types_present <- function(df_to_plot, plural_geo_title) {
   samples_per_day = df_to_plot %>%
     group_by(time_value) %>%
     summarize(n = n())
   
+  day_diff =
+    samples_per_day %>% 
+    summarise(difftime(max(time_value), min(time_value), units = "days"))
```

Comment:
...is this not just the full time span this data is available?

Reply:
It's the range of time for the signal that was retrieved (e.g, if we requested 28 days worth of data and get back 25 due to 3 day lag, this is = 25), i.e., this makes the plotting functions extract the range from whatever signal data is passed in, rather than assuming any specific value.